modules/cluster-logging-elasticsearch-retention.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-elasticsearch.adoc
+
+[id="cluster-logging-elasticsearch-retention_{context}"]
+= Configuring Elasticsearch retention policy
+
+
+Each Elasticsearch index adds load and overhead to the Elasticsearch cluster. You can use the Elasticsearch rollover index API to automatically rollover indices when they reach a specified age. When an index reaches the designated age, Elasticsearch rolls over the index, moving the current index and creating a new index. After seven days, the old index is automatically deleted. Rolling over and deleting indices optimizes the indices and helps Elasticsearch performance. 
+
+You specify how long Elasticsearch stores indices using a separate _retention policy_ for each of the three log sources, infrastucture logs, application logs, and audit logs, using the Cluster Logging Custom Resource (CR) to set the number of days after which the indices should be deleted. The Cluster Logging CR writes the full policy to the Elasticsearch Custom Resource (CR).
+
+////
+Only maxAge in current implementation:
+You can specify any of the following conditions:
+* `max_age`. The maximum age that is allowed before triggering a rollover for the _hot_ indices, or those ​indices that are actively being updated and queried. Specify a value in days (d), hours (h), minutes (m), or seconds (s). 
+* `max_docs`. The maximum number of documents allowed in an index before triggering a rollover. Documents added since the last refresh are not included in the document count. The document count does not include documents in replica shards.
+* `max_size`. The maximum size the index can be before a rollover is triggered. Specify a value in bytes (b), kilobytes (kb), megabytes (mb), or gigabytes (gb). 
+////
```

Comment:
Do we need this block? we support only maxAge.

Reply:
@periklis I commented-this section out. Wasn't sure if it was needed. I can remove.